src/dosbox.cpp

```diff
@@ -1222,6 +1222,10 @@ void DOSBOX_Init()
 	        "tab-separated format, used by SETVER.EXE as a persistent storage\n"
 	        "(empty by default).");
 
+	pstring = secprop->Add_path("cmd_history_file", only_at_start, "cmd_history.txt");
+	pstring->Set_help(
+		"File containing persistent command line history ('cmd_history.txt' by default)");
```

Comment:
minor: missing dot at the end of the sentence.

Reply:
"File where the persistent command

Reply:
Maybe call it `command_line_history_file`?

Reply:
Yeah that name looks better for the config setting.  Fixed.